src/table/TableWrapper.jsx

```diff
@@ -21,16 +21,28 @@ const useStyles = makeStyles({
   paginationHidden: {
     display: 'none',
   },
+  containerMobileLayout: {
+    height: 'calc(100% - 120px)',
+  },
+  paginationMobileLayout: {
+    marginTop: '-15px',
+    '& div:nth-of-type(1)': {
+      display: 'initial',
+    },
+  },
 });
 
 export default function TableWrapper(props) {
   const { tableData, setPageInfo, constraints } = props;
   const { size, rows } = tableData;
+  const [tableWidth, setTableWidth] = useState();
   const [page, setPage] = useState(0);
   const [rowsPerPage, setRowsPerPage] = useState(100);
   const classes = useStyles();
   const containerMode = constraints.active ? 'containerOverflowHidden' : 'containerOverflowAuto';
   const clientMode = constraints.active && 'paginationHidden';
+  const containerLayout = tableWidth < 380 && 'containerMobileLayout';
```

Comment:
is 380 where sense-client goes into mobile mode?

Reply:
it is 380px that the next page button starts to become invisible

Reply:
But why 380?

Reply:
I just change the window size and when the window size is 380px, the next page button starts to become invisible

Reply:
okok

Reply:
Changed to 400 since it gets a bit tight if you hav lik 100000 rows, the `1-100 of 100000` string becomes long